fetch/client/src/main/scala/endpoints4s/fetch/EndpointsSettings.scala

```diff
@@ -2,39 +2,44 @@ package endpoints4s.fetch
 
 import endpoints4s.Hashing
 
-import scala.annotation.nowarn
+import scala.concurrent.duration.FiniteDuration
 
 /** Settings for XHR interpreter.
   * @param baseUri Base of the URI of the service that implements the endpoints, can be absolute or relative (e.g. "http://foo.com" or "/bar")
   */
-final class EndpointsSettings private (val baseUri: Option[String]) extends Serializable {
+final class EndpointsSettings private (val baseUri: Option[String], val timeout: Option[FiniteDuration]) extends Serializable {
```

Comment:
To keep the backward binary compatibility, you need to add back the previous constructor as a secondary constructor:

```suggestion
final class EndpointsSettings private (val baseUri: Option[String], val timeout: Option[FiniteDuration]) extends Serializable {

  def this(baseUri: Option[String]) = this(baseUri, None)
```